l5kit/l5kit/rasterization/satellite_rasterizer.py

```diff
@@ -41,7 +41,11 @@ def __init__(
         self.map_pixel_scale = (1 / np.linalg.norm(map_to_sat[0, 0:3]) + 1 / np.linalg.norm(map_to_sat[1, 0:3])) / 2
 
     def rasterize(
-        self, history_frames: np.ndarray, history_agents: List[np.ndarray], agent: Optional[np.ndarray] = None
+        self,
```

Comment:
the interface is the same one because we call it internally in our code. Putting a warning wouldn't help the user I think

Reply:
What I mean here is that until now there was a feature parity between semantic map and satellite map. Now the TLs introduce quite a significant change to that. Let's make sure we document that the satellite map does not support TLs. Otherwise, people training on sat map and sem map might not know the difference and be surprised of the results.